docs/developer-docs/integrations/ledger/icp-index-local-setup.md

```diff
@@ -0,0 +1,233 @@
+Which will return a vector the encoded blocks.
+```
+(
+  record {
+    blocks = vec {
+      blob "\12\0a\08\90\83\a6\c6\c7\b8\a6\c3\17\1a=\12-\12\22\0a \0amCu\816\9bTj\fd\efa<\a9\c0\81\a2R\ca,F\e7\ec)\e5\10\bc\10\b2\13\fa\27\1a\07\08\80\d0\db\c3\f4\02\22\002\0a\08\90\83\a6\c6\c7\b8\a6\c3\17";
+      blob "\0a\22\0a \912w)\02\f3a\9e\bc+\eax\e8D\b9\c9\09\14\12\cc%ZNRJ\06\c7?\a8\d1\97/\12\0a\08\a8\cd\b5\fb\a6\ba\a6\c3\17\1aW\1aS\0a\22\0a \0amCu\816\9bTj\fd\efa<\a9\c0\81\a2R\ca,F\e7\ec)\e5\10\bc\10\b2\13\fa\27\12\22\0a \930\d6\d8\cd\8ar\a5\a9Z\b7\d6@P\18\c4\ca^\bd\0cN\c1o6\eb\91\dbu\14\bd\86#\1a\04\08\a0\8d\06\22\03\08\90N\22\00";
+    };
+    chain_length = 2 : nat64;
+  },
+)
+```
+
+To fetch a vector of all transactions your default account was involved in you can use the following commands.
+Find out the principal of your default account.
+```
+dfx identity get-principal
+```
+In the case of this tutorial this returns
+
+```
+hdq6b-ncywm-yajd5-4inc6-hgpzp-55xnp-py7d5-uqt6o-cv5c6-rrhwa-zqe
+```
+
+Then you can query the transactions for this principal with the default subaccount set by calling. 
+```
+dfx canister call qhbym-qaaaa-aaaaa-aaafq-cai get_account_transactions '(record{account=record {owner = principal "hdq6b-ncywm-yajd5-4inc6-hgpzp-55xnp-py7d5-uqt6o-cv5c6-rrhwa-zqe"}; max_results=2:nat})'
+```
+
+The result will include the intial mint operation as well as the transfer that we made.
+
+```
+(
+  variant {
+    Ok = record {
+      balance = 99_999_890_000 : nat64;
+      transactions = vec {
+        record {
+          id = 1 : nat64;
+          transaction = record {
+            memo = 0 : nat64;
+            icrc1_memo = null;
+            operation = variant {
+              Transfer = record {
+                to = "9330d6d8cd8a72a5a95ab7d6405018c4ca5ebd0c4ec16f36eb91db7514bd8623";
+                fee = record { e8s = 10_000 : nat64 };
+                from = "0a6d437581369b546afdef613ca9c081a252ca2c46e7ec29e510bc10b213fa27";
+                amount = record { e8s = 100_000 : nat64 };
+              }
+            };
+            created_at_time = null;
+          };
+        };
+        record {
+          id = 0 : nat64;
+          transaction = record {
+            memo = 0 : nat64;
+            icrc1_memo = null;
+            operation = variant {
+              Mint = record {
+                to = "0a6d437581369b546afdef613ca9c081a252ca2c46e7ec29e510bc10b213fa27";
+                amount = record { e8s = 100_000_000_000 : nat64 };
+              }
+            };
+            created_at_time = opt record {
+              timestamp_nanos = 1_695_211_378_870_682_000 : nat64;
+            };
+          };
+        };
+      };
+      oldest_tx_id = opt (0 : nat64);
+    }
+  },
+)
+```
+The ICP ledger uses AccountIdentifier which are a hashed version of Account (Principal and Subaccount) for privacy. This also means that the returned transactions will show accounts as the hash in bytes rather than the actual Accounts.
```

Comment:
```suggestion
The ICP ledger uses `AccountIdentifier` which are a hashed version of `Account` (`Principal` and `Subaccount`) for privacy. This also means that the returned transactions will show accounts as the hash in bytes rather than the actual Accounts.
```